hazelcast-client/src/main/resources/hazelcast-client-config-3.8.xsd

```diff
@@ -349,10 +349,22 @@
 
     <xs:complexType name="near-cache">
         <xs:all>
-            <xs:element name="max-size" type="xs:unsignedInt" minOccurs="0" maxOccurs="1" default="0"/>
+            <xs:element name="max-size" type="xs:unsignedInt" minOccurs="0" maxOccurs="1" default="0">
+                <xs:annotation>
+                    <xs:documentation>
+                        Deprecated since 3.8, please use <eviction/>
```

Comment:
That's a very good question, I have little experience with XSD. A `{@link ...}` would be great, but I doubt that it exists.